lib/index.js

```diff
@@ -30,6 +31,15 @@ const packageFileNames = {
 const stdinWarningTime = 5000
 const stdinWarningMessage = `Hmmmmm... this is taking a long time. Your console is telling me to wait for input \non stdin, but maybe that is not what you want.\nTry ${chalk.cyan('winpty ncu.cmd')}, or specify a package file explicitly with ${chalk.cyan('--packageFile package.json')}. \nSee https://github.com/raineorshine/npm-check-updates/issues/136#issuecomment-155721102`
 
+// In the format of [argName, deprecationMessage]
+const deprecatedArgs = [
+  ['greatest', '--greatest has been deprecated. Renamed to "--target greatest"'],
+  ['newest', '--newest has been deprecated. Renamed to "--target newest".'],
+  ['o', '-o has been deprecated. Renamed to "--output ownerChanged".'],
+  ['ownerChanged', '--ownerChanged has been deprecated. Renamed to "--output ownerChanged".'],
+  ['semverLevel', '--semverLevel has been deprecated. Renamed to --target']
+]
```

Comment:
Thanks for adding these! Let's put them directly in `cli-options` so we don't have to maintain parallel data structures. You can add a `deprecated?: string` property to each option and then filter them later to print them out.

Reply:
Definitely!